lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingTransportClient.cs

```diff
@@ -98,17 +97,13 @@ public void Send(T message)
         /// </summary>
         public void Dispose()
         {
-            Dispose(true);
-            GC.SuppressFinalize(this);
-        }
-
-        protected void Dispose(bool disposing)
-        {
-            if (!_disposed && disposing)
+            if (!_disposed)
             {
                 _link.Dispose();
                 _disposed = true;
             }
+
+            GC.SuppressFinalize(this);
```

Comment:
Removed the destructor.

On Tue, Jul 7, 2015 at 12:49 PM, Markus Weimer <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingTransportClient.cs
> <https://github.com/apache/incubator-reef/pull/261#discussion_r34082143>:
>
> >              {
> >                  _link.Dispose();
> >                  _disposed = true;
> >              }
> > +
> > +            GC.SuppressFinalize(this);
>
> Why do we need to do this? This type doesn't even have a destructor.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/261/files#r34082143>.
>